scripts/build_ios.sh

```diff
@@ -9,11 +9,24 @@
 
 CAFFE2_ROOT="$( cd "$(dirname "$0")"/.. ; pwd -P)"
 
+if [ -z "$PYTHON" ]; then
```

Comment:
Do you need to have this check for PYTHON here?  AFAICT, as long as you run the script with `${CONDA_RUN}`, python is there.  May be only need to change python to python3 to make sure that python3 is used?

Reply:
That did seem like a suitable fail-safe, which would support the use of a custom python path if needed while also checking that without specific path, python3 is used.
I've tested with `${CONDA_RUN}` though `$PYTHON` remains undefined. I can leave it as is, or remove the custom path feature and always use a default `python[3]`. What would you prefer?

Reply:
I'm not strongly flavor one over another, so it's fine to keep what you have here as it's proven to be working fine